docs/guides/developing-components/codelabs.md

```diff
@@ -10,18 +10,18 @@ A quick primer on the basics of web components. Great if you're just getting sta
   > Go to the codelab
 </a>
 
-## lit-html & lit-element: basics
+## lit: basics
```

Comment:
Should it be `lit` or `Lit`?

Reply:
The team seems to use `Lit` throughout their site.

Reply:
Updated. I tried to be thorough but with 1500+ search results for "lit" I might have missed a few places :P